hardware/tests/test_detect.py

```diff
@@ -156,6 +156,7 @@ def test_get_cpus(self, mock_output_lines):
                 side_effect=[
                     sample('lscpu-vm').split('\n'),
                     sample('lscpu-vmx').split('\n'),
+                    sample('lscpu-vmrhel8').split('\n'),
```

Comment:
Sorry for confusion, I think we actually need a new test case instead of updating this one.

Reply:
Can you expand a bit as to how you want this to look ?

Reply:
I actually reproposed without the test for now as the fix is very simple and I just got a completed IPA introspection on a rhel8 based IPA initramfs against a RHEL8 VM. (am a bit swamped in a bunch of other bugs, so would prefer if we can merge this as is at this point. hope that is okay?)

Reply:
fine by me